### DISCUSSION TITLE
Connecting to Postgres with new user

### DISCUSSION THREAD

**Discussion by denonrailz** [supabase/supabase#23759] [Questions]
Hi,
I'm plan to work with supabase with my golang backend service through direct Postgres connection.
In case of security I create new Postgres user and provide limited access in case of my business logic. I have just provide SELECT and UPDATE, and CONNECTION

Now I have an error while direct connection to Postgres:
`server error (FATAL: Tenant or user not found (SQLSTATE XX000))`

connection with default Postgres user works fine:
`user=postgres.vkp... password= host=aws-0-eu-central-1.pooler.supabase.com port=5432 dbname=postgres`

This not works
backend_user.vkp... password= host=aws-0-eu-central-1.pooler.supabase.com port=5432 dbname=postgres

What does I do wrong?

**Comment by GaryAustin1** [comment]
Is your new user setup to login and have a password?

**Reply by mikeburgh** [reply]
I have a smiliar issue.. but it seems related to a delay after creating/changing the users password.

If I create the user and attempt to connect straight away, or after the users password is changed.. it does not work... Give it anywhere from 1 to 3 minutes.. and it works.

I assume there is some sort of sync between the connection pooler and the postgres user table ?

**Comment by ravigill3969** [comment]
if u got the solution, please post 
I am stuck on this part as wll.

**Reply by apensotti** [reply]
Did you ever find a solution. I am having this problem.

**Reply by ravigill3969** [reply]
no bro, I went with mongoDB and now I am working on project in which I will use it but only after it completes, less see how it goes.
sorry for too late reply.
